build.gradle

```diff
@@ -117,12 +117,13 @@ dependencies {
         exclude group: 'xerces', module: 'xmlParserAPIs'
         exclude group: 'xerces', module: 'xercesImpl'
     }
-    compile('org.apache.httpcomponents:httpclient:4.4.1') {
+    compile('org.apache.httpcomponents:httpclient:4.5.13') {
         exclude group: 'commons-logging', module: 'commons-logging'
     }
-    compile 'org.apache.httpcomponents:httpcore:4.4.1'
+    compile 'org.apache.httpcomponents:httpcore:4.4.14'
+    compile 'org.apache.commons:commons-compress:1.21'
```

Comment:
i. Can we remove adding this direct dependency. 
```suggestion

```
ii. If the vulnerability of using earlier versions of commons-compress cannot be ignored, then suggest we add like this
```
    compile 'nl.javadude.scannit:scannit:1.4.1'
    constraints {
        compile('org.apache.commons:commons-compress:1.21') {
            because 'earlier versions of this transitive dependency of scannit have a vulnerability issue'
        }
    } 
```